### ISSUE TITLE
Some Numpy false positives

### ISSUE THREAD

**Issue by Apromixately** [PyCQA/pylint#3208]
### Steps to reproduce
Sorry for putting these in one issue but I am guessing that they are probably small issues.

I am getting `Assigning to function call which doesn't return (assignment-from-no-return)` for 
`foo = np.bitwise_and(bar, doom)` and the same for `np.bitwise_xor`.

I am also getting `Instance of 'tuple' has no 'resize' member (no-member)` for a return value from `np.bincount`. `np.bincount` does not return tuples.

### pylint --version output
No config file found, using default configuration
__main__.py 1.9.5,
astroid 1.6.6
Python 2.7.15+ (default, Jul  9 2019, 16:51:35)
[GCC 7.4.0]

**Comment by PCManticore** [comment]
Hey @Apromixately Thanks for opening an issue. Is it possible to try with a more recent version of `pylint` such as 2.4 and astroid 2.3+? These should have a ton of `numpy` improvements and some or all these issues might not reproduce any longer.

Also it would be great to edit the description and show a small snippet that can be used to reproduce the corresponding issue.

**Comment by Apromixately** [comment]
The bitwise_and and bitwise_xor problems have disappeared in
```
pylint 2.4.3
astroid 2.3.3
Python 3.7.5rc1 (default, Oct  8 2019, 16:47:45) 
[GCC 9.2.1 20191008]
```
However, I don't know if those can be used with Python 2. They don't seem to be in PyPi. Then again [Python 2 is maybe not important anymore.](https://pythonclock.org/)

The bincount problem persists.

The following snipped reproduces all problems with Python 2 Pylint and the bincount problem with Python 3 pylint.

```
import numpy as np

def main():
    first_array = np.array([15, 7, 8, 3, 4])
    second_array = np.array([7])
    bitwise_xor = np.bitwise_xor(first_array, second_array)
    print(bitwise_xor)
    bitwise_and = np.bitwise_and(first_array, second_array)
    print(bitwise_and)
    count = np.bincount(first_array)
    print(count)
    count.resize(20)
    print(count)

if __name__ == "__main__":
    main()
```

**Comment by hippo91** [comment]
@Apromixately thanks for your snippet. I can confirm the bug with `bincount`. I'll try to fix it ASAP.

**Comment by texadactyl** [comment]
Running pylint 2.4.4 on this Python 3 script:

```
"""
power versus float_power
"""
import numpy as np

XX1 = np.power(np.pi, 2)
XX2 = np.float_power(np.pi, 2)
print(XX1, XX2)
```
I believe that pylint misdiagnoses the XX2 assignment.  There is, indeed, a float_power function in numpy.  Maybe it presents a different function definition in numpy that is confusing?
```
************* Module essai
essai.py:7:6: E1101: Module 'numpy' has no 'float_power' member (no-member)
```

Execution of the script in python3 produces the expected result:
```
9.869604401089358 9.869604401089358
```

**Comment by hippo91** [comment]
@Apromixately 
Your problem should be solved now, thanks to the PR : PyCQA/astroid#734.

@texadactyl could you please open another issue for the `float_power` problem. This function is not defined in the same module as those encountered by @Apromixately. Thanks in advance.
